openpype/plugins/publish/validate_sequence_frames.py

```diff
@@ -7,28 +12,51 @@ class ValidateSequenceFrames(pyblish.api.InstancePlugin):
     The files found in the folder are checked against the startFrame and
     endFrame of the instance. If the first or last file is not
     corresponding with the first or last frame it is flagged as invalid.
+
+    Used regular expression pattern handles numbers in the file names
+    (eg "Main_beauty.v001.1001.exr", "Main_beauty_v001.1001.exr",
+    "Main_beauty.1001.1001.exr") but not numbers behind frames (eg.
+    "Main_beauty.1001.v001.exr")
     """
 
     order = pyblish.api.ValidatorOrder
     label = "Validate Sequence Frames"
-    families = ["imagesequence"]
-    hosts = ["shell"]
+    families = ["imagesequence", "render"]
+    hosts = ["shell", "unreal"]
 
     def process(self, instance):
-
-        collection = instance[0]
-        self.log.info(collection)
-
-        frames = list(collection.indexes)
-
-        current_range = (frames[0], frames[-1])
-        required_range = (instance.data["frameStart"],
-                          instance.data["frameEnd"])
-
-        if current_range != required_range:
-            raise ValueError("Invalid frame range: {0} - "
-                             "expected: {1}".format(current_range,
-                                                    required_range))
-
-        missing = collection.holes().indexes
-        assert not missing, "Missing frames: %s" % (missing,)
+        representations = instance.data.get("representations")
+        if not representations:
+            return
+        for repr in representations:
+            repr_files = repr["files"]
+            if isinstance(repr_files, str):
```

Comment:
Question 1: Do we want to do it for every representation? What if there is thumbnail?
Question 2: If files has string, it will 100% crash because of remainders.